lib/addax-rdbms/src/main/java/com/wgzhao/addax/rdbms/reader/CommonRdbmsReader.java

```diff
@@ -272,7 +272,12 @@ protected Column createColumn(ResultSet rs, ResultSetMetaData metaData, int i)
                     return new DateColumn(rs.getDate(i));
 
                 case Types.TIMESTAMP:
-                    return new TimestampColumn(rs.getTimestamp(i, Calendar.getInstance()));
+                    if(!"org.apache.hive.jdbc.HiveDriver".equals(this.dataBaseType.getDriverClassName())){
```

Comment:
In the `hivereader` plugin, you can achieve specific handling for reading the Timestamp type by overriding the `createColumn` method. You can refer to the implementation in the following link for guidance:https://github.com/wgzhao/Addax/blob/master/plugin/reader/postgresqlreader/src/main/java/com/wgzhao/addax/plugin/reader/postgresqlreader/PostgresqlReader.java

Reply:
Because it needs to be encapsulated into a unified script, hivereader will not be used, but rdbmsreader will be used instead